keras_cv/models/vgg19.py

```diff
@@ -0,0 +1,176 @@
+# Copyright 2022 The KerasCV Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+"""VGG19 model for KerasCV.
+
+Reference:
+  - [Very Deep Convolutional Networks for Large-Scale Image Recognition](
+      https://arxiv.org/abs/1409.1556) (ICLR 2015)
+"""
+
+import tensorflow as tf
+from tensorflow import keras
+from tensorflow.keras import layers
+
+
+def VGG19(
+    include_rescaling,
+    include_top,
+    num_classes=None,
+    weights=None,
+    input_shape=(None, None, 3),
+    pooling=None,
+    classifier_activation="softmax",
+    name="VGG19",
+    **kwargs,
+):
+    """Instantiates the VGG19 architecture.
+
+    Reference:
+    - [Very Deep Convolutional Networks for Large-Scale Image Recognition](
```

Comment:
docstrings can have links on one line now.  I updated the lint rules to support this, so feel free to put this in a single line.  It breaks keras.io to have it on multiple lines in some circumstances.

Reply:
Done